src/EditorFeatures/VisualBasic/Formatting/Indentation/VisualBasicIndentationService.Indenter.vb

```diff
@@ -41,10 +41,20 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.Formatting.Indentation
                         Return GetIndentationOfLine(previousLine)
                     End If
 
-                    If trivia.Kind = SyntaxKind.LineContinuationTrivia OrElse trivia.Kind = SyntaxKind.CommentTrivia Then
+                    If trivia.Kind = SyntaxKind.LineContinuationTrivia Then
                         Return GetIndentationBasedOnToken(GetTokenOnLeft(trivia), trivia)
                     End If
 
+                    If trivia.Kind = SyntaxKind.CommentTrivia Then ' Two cases comment or _ followed by comment
+                        Dim FirstTrivia As SyntaxTrivia = Tree.GetRoot(CancellationToken).FindTrivia(token.Span.End + 1)
```

Comment:
>FindTrivia(token.Span.End + 1) [](start = 90, length = 30)

> FindTrivia(token.Span.End + 1)
I didn't quite follow this logic. Could you clarify?

Reply:
The original code looked for an _ as the last trivial before newline and then indented the next line correctly but with the new logic the last trivia before the newline could be a comment. Lines 49-56 handle this. GetIndentationBasedOnToken handles a ending comment different then an _ but now we want _ Comment handled like _. I will add more comments to help clarify.